the-exterminators/error-annihilator
# Sp 62 view single ticket

## Linked Issues

## PR Description
This view is where you land when you enter a ticket number in the search bar. It is far from done, it still needs work after the database is ready.

## Changes


## How to test

1. fetch branch
2. Open, build and run project
3. Check if project builds and looks correct on first sight
4. Open the header and enter a number into the search bar. Hit enter. The single ticket view should open up with the entered number in the URL and title
5. The form is prefilled with in a memory ticket so dont expect it to change
6. You can enter another ticket number and it changes the title and url
7. Refreshing the page doesnt work yet - I will fix this after the database is ready
8. You also cant enter the same number twice, because with the in memory stuff I am always creating new Tickets and it doesnt like duplicates! This will change with the database implementation.

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
